dextra-components/src/components/Plot/src/Plot.ts

```diff
@@ -54,6 +54,17 @@ export class OslPlot extends OslData {
   @property({ type: Number })
   height = 400;
 
+  @property({ converter: interpretFuncJsonOrString })
+  x_axis?: Plot.AxisOptions["anchor"] | boolean | null;
```

Comment:
We should use pascalCase on all properties, as snake_case does not conform to some javascript style standards

Reply:
[21aca46](https://github.com/open-spatial-lab/dextra/pull/4/commits/21aca468057fb1f2d8a798507653fba84b1df7bd)
Changed to 'xAxisAnchor'/'yAxisAnchor' and 'xAxisScaling'/'yAxisScaling'